WordPress/Classes/ViewRelated/Stats/Insights/StatsTotalInsightsCell.swift

```diff
@@ -50,6 +52,45 @@ struct StatsTotalInsightsData {
 
         return sparklineData
     }
+
+    public static func makeTotalInsightsGuideText(insightsStore: StatsInsightsStore, statsSummaryType: StatsSummaryType) -> NSAttributedString? {
+        switch statsSummaryType {
+        case .likes:
+            guard let summary = insightsStore.getLastPostInsight() else {
+                return nil
+            }
+
+            let formattedText: String
+            if summary.likesCount == 1 {
+                formattedText = TextContent.likesTotalGuideTextSingular
+            } else {
+                formattedText = String(format: TextContent.likesTotalGuideTextPlural, summary.title, summary.likesCount)
+            }
+
+            return NSAttributedString.attributedStringWithHTML(formattedText, attributes: StatsTotalInsightsData.guideAttributes)
+        case .comments:
+            return NSAttributedString(string: TextContent.commentsTotalGuideText)
+        default:
+            return nil
+        }
+    }
+
+    private static var guideAttributes: StyledHTMLAttributes = [
+        .BodyAttribute: [
+            .font: UIFont.preferredFont(forTextStyle: .subheadline),
+            .foregroundColor: UIColor.text
+        ],
+            .ATagAttribute: [
+                .foregroundColor: UIColor.primary,
+                .underlineStyle: 0
+            ]
```

Comment:
indent is off here - I'm sure its one of those Xcode "quirks" where it looks fine in Xcode